vm/vm.go

```diff
@@ -297,7 +305,19 @@ func (vm *VM) Run(program *Program, env interface{}) (_ interface{}, err error)
 		case OpCall:
 			fn := reflect.ValueOf(vm.pop())
 			size := arg
-			in := make([]reflect.Value, size)
+			fnt := fn.Type()
+			var (
+				args []reflect.Value
+				in   []reflect.Value
+			)
+			if fnt.NumIn() > 0 && fn.Type().In(0) == contextType {
```

Comment:
Let's not introduce context passing for general OpCall. 

Let's allow context to be passed only to funcs defined via `expr.FunctionWithContext` in OpCallContext.